--- origsrc/poppler-0.10.1/configure.ac	2008-11-09 14:12:41.000000000 -0600
+++ src/poppler-0.10.1/configure.ac	2008-11-09 19:56:27.992013500 -0600
@@ -27,6 +27,9 @@
 win32_libs=""
 create_shared_lib=""
 case "$host_os" in
+  cygwin*)
+    create_shared_lib="-no-undefined"
+  ;;
   mingw*)
     win32_libs="-lgdi32"
     create_shared_lib="-no-undefined"
--- origsrc/poppler-0.10.1/qt/Makefile.am	2008-10-09 15:30:34.000000000 -0500
+++ src/poppler-0.10.1/qt/Makefile.am	2008-11-09 19:56:49.632638500 -0600
@@ -29,7 +29,7 @@
 	$(FREETYPE_LIBS)			\
 	$(FONTCONFIG_LIBS)
 
-libpoppler_qt_la_LDFLAGS = -version-info 2:0:0
+libpoppler_qt_la_LDFLAGS = -version-info 2:0:0 @create_shared_lib@
 
 if BUILD_SPLASH_OUTPUT
 
--- origsrc/poppler-0.10.1/qt4/src/Makefile.am	2008-10-09 15:30:34.000000000 -0500
+++ src/poppler-0.10.1/qt4/src/Makefile.am	2008-11-09 19:57:02.929513500 -0600
@@ -61,7 +61,7 @@
 	$(top_builddir)/poppler/libpoppler-arthur.la
 endif
 
-libpoppler_qt4_la_LDFLAGS = -version-info 4:0:1
+libpoppler_qt4_la_LDFLAGS = -version-info 4:0:1 @create_shared_lib@
 
 # This rule lets GNU make create any *.moc from the equivalent *.h
 .h.moc:
