--- origsrc/poppler-0.12.0/configure.ac	2009-09-23 04:35:47.586377400 -0500
+++ src/poppler-0.12.0/configure.ac	2009-09-23 04:34:44.545414100 -0500
@@ -43,6 +43,9 @@ win32_libs=""
 create_shared_lib=""
 auto_import_flags=""
 case "$host_os" in
+  cygwin*)
+    create_shared_lib="-no-undefined"
+  ;;
   mingw*)
     win32_libs="-lgdi32"
     create_shared_lib="-no-undefined"
--- origsrc/poppler-0.12.0/qt/Makefile.am	2009-09-23 04:35:47.591377400 -0500
+++ src/poppler-0.12.0/qt/Makefile.am	2009-09-23 04:34:44.000000000 -0500
@@ -30,7 +30,7 @@ libpoppler_qt_la_LIBADD = 			\
 	$(FREETYPE_LIBS)			\
 	$(FONTCONFIG_LIBS)
 
-libpoppler_qt_la_LDFLAGS = -version-info 2:0:0
+libpoppler_qt_la_LDFLAGS = -version-info 2:0:0 @create_shared_lib@
 
 if BUILD_SPLASH_OUTPUT
 
--- origsrc/poppler-0.12.0/qt4/src/Makefile.am	2009-09-23 04:35:47.595127500 -0500
+++ src/poppler-0.12.0/qt4/src/Makefile.am	2009-09-23 04:34:44.000000000 -0500
@@ -61,7 +61,7 @@ libpoppler_qt4_la_LIBADD += 			\
 	$(top_builddir)/poppler/libpoppler-arthur.la
 endif
 
-libpoppler_qt4_la_LDFLAGS = -version-info 5:0:2
+libpoppler_qt4_la_LDFLAGS = -version-info 5:0:2 @create_shared_lib@
 
 # This rule lets GNU make create any *.moc from the equivalent *.h
 .h.moc:
