--- origsrc/poppler-0.18.1/poppler-glib.pc.cmake	2011-10-17 06:42:29.000000000 -0500
+++ src/poppler-0.18.1/poppler-glib.pc.cmake	2011-11-17 02:20:29.832764800 -0600
@@ -6,7 +6,7 @@ includedir=${prefix}/include
 Name: poppler-glib
 Description: GLib wrapper for poppler
 Version: @POPPLER_VERSION@
-Requires: glib-2.0 >= $GLIB_REQUIRED gobject-2.0 >= $GLIB_REQUIRED cairo >= $CAIRO_VERSION
+Requires: glib-2.0 >= @GLIB_REQUIRED@ gobject-2.0 >= @GLIB_REQUIRED@ cairo >= @CAIRO_VERSION@
 @PC_REQUIRES_PRIVATE@
 
 Libs: -L${libdir} -lpoppler-glib
