--- origsrc/poppler-0.8.4/configure.ac	2008-06-28 10:17:09.000000000 -0500
+++ src/poppler-0.8.4/configure.ac	2008-07-08 04:15:45.281250000 -0500
@@ -27,6 +27,9 @@
 win32_libs=""
 create_shared_lib=""
 case "$host_os" in
+  cygwin*)
+    create_shared_lib="-no-undefined"
+  ;;
   mingw|mingw32)
     win32_libs="-lgdi32"
     create_shared_lib="-no-undefined"
--- origsrc/poppler-0.8.4/glib/Makefile.am	2008-03-26 14:38:52.000000000 -0500
+++ src/poppler-0.8.4/glib/Makefile.am	2008-07-08 04:15:45.296875000 -0500
@@ -94,7 +94,7 @@
 	$(FONTCONFIG_LIBS)				\
 	$(cairo_libs)
 
-libpoppler_glib_la_LDFLAGS = -version-info 3:0:0
+libpoppler_glib_la_LDFLAGS = -version-info 3:0:0 @create_shared_lib@
 
 if BUILD_WITH_GDK
 noinst_PROGRAMS = test-poppler-glib
--- origsrc/poppler-0.8.4/qt/Makefile.am	2008-03-26 14:38:52.000000000 -0500
+++ src/poppler-0.8.4/qt/Makefile.am	2008-07-08 04:15:45.328125000 -0500
@@ -29,7 +29,7 @@
 	$(FREETYPE_LIBS)			\
 	$(FONTCONFIG_LIBS)
 
-libpoppler_qt_la_LDFLAGS = -version-info 2:0:0
+libpoppler_qt_la_LDFLAGS = -version-info 2:0:0 @create_shared_lib@
 
 if BUILD_SPLASH_OUTPUT
 
--- origsrc/poppler-0.8.4/qt4/src/Makefile.am	2008-06-08 16:41:11.000000000 -0500
+++ src/poppler-0.8.4/qt4/src/Makefile.am	2008-07-08 04:15:45.343750000 -0500
@@ -59,7 +59,7 @@
 	$(top_builddir)/poppler/libpoppler-arthur.la
 endif
 
-libpoppler_qt4_la_LDFLAGS = -version-info 3:0:0
+libpoppler_qt4_la_LDFLAGS = -version-info 3:0:0 @create_shared_lib@
 
 # This rule lets GNU make create any *.moc from the equivalent *.h
 .h.moc:
--- origsrc/poppler-0.8.4/test/Makefile.am	2008-03-26 14:38:52.000000000 -0500
+++ src/poppler-0.8.4/test/Makefile.am	2008-07-08 04:15:45.359375000 -0500
@@ -68,8 +68,8 @@
        pdf-inspector.cc
 
 pdf_inspector_LDADD =					\
-	$(top_builddir)/poppler/libpoppler.la		\
 	$(top_builddir)/poppler/libpoppler-cairo.la	\
+	$(top_builddir)/poppler/libpoppler.la		\
 	$(CAIRO_LIBS)					\
 	$(FREETYPE_LIBS)				\
 	$(GTK_TEST_LIBS)
